src/Command/Pull/PullCommandBase.php

```diff
@@ -47,6 +47,11 @@ abstract class PullCommandBase extends CommandBase {
    */
   protected $drushHasActiveDatabaseConnection;
 
+  /**
+   * @var string
+   */
+  private $site;
```

Comment:
FYI at Acquia we typically try to refer to an application, not a site. I however understand for the purpose of what we're trying to achieve here (pulling a specific site in the IDE), `site` might work better.